src/components/Todos/index.js

```diff
@@ -7,15 +7,27 @@ import deserialize from './storage-source';
 import serialize from './storage-sink';
 import TodoItem from '../TodoItem';
 
+// AMEND STATE WITH CHILDREN
+// This function creates the projection function
+// for the map function below.
 function amendStateWithChildren(DOMSource) {
   return function (todosData) {
     return {
       ...todosData,
+      // The list property is the only one being amended.
+      // We map over the array in the list property to
+      // enhance them with the actual todo item data flow components.
       list: todosData.list.map(data => {
+        // Turn the data item into an Observable
         const props$ = Observable.just(data);
+        // Create scoped todo item data flow components.
```

Comment:
Would be good if `data flow` is `dataflow`, and this line should say "component" not "components".